docs/sources/operations/storage/retention.md

```diff
@@ -8,7 +8,8 @@ Retention in Loki is achieved either through the [Table Manager](#table-manager)
 
 Retention through the [Table Manager](../table-manager/) is achieved by relying on the object store TTL feature, and will work for both [boltdb-shipper](../boltdb-shipper) store and chunk/index store. However retention through the [Compactor](../boltdb-shipper#compactor) is supported only with the [boltdb-shipper](../boltdb-shipper) store.
 
-The [Compactor](#Compactor) retention will become the default and have long term support. While this retention is still **experimental**, it supports more granular retention policies on per tenant and per stream use cases.
+<span style="background-color:#f3f973;">Retention through the [Compactor](#Compactor) is experimental.</span>
```

Comment:
Github markdown does not support setting the background colour for text. This would only work for docs hosted on Grafana.
People reading the docs on the Github repo would see it in plain text https://github.com/grafana/loki/blob/79ed6ffdea5d66935f6adeac4abadfcc9cce3624/docs/sources/operations/storage/retention.md

There are some suggestions there to use a `diff` block to highlight the text in this issue https://github.com/github/markup/issues/1440

Reply:
You are correct.  I am only doing this for hosted docs. And, this is a temporary solution. I just wanted something quick and easily removable for the upcoming 2.3.0 release.

What do you think about using this as a temporary and partial fix until we decide if there is a more permanent way of solving what I think is a readability issue, @sandeepsukhani ?